src/api.js

```diff
@@ -7,4 +7,9 @@ const newsApi = axios.create({
 export const getArticles = async () => {
     const response = await newsApi.get('/articles')
     return response.data;
+}
+
```

Comment:
Nice to see you have a base URL which can be used for all your request functions 👍